101-backup-la-reporting/nestedtemplates/azure_alerts.json

```diff
@@ -0,0 +1,547 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "workspaceLocation":{
+         "type":"string",
+         "metadata":{
+            "description":"Specify the workspace region"
+         }
+      },
+      "workspaceName":{
+         "type":"string",
+         "metadata":{
+            "description":"Specify the workspace name"
+         }
+      }
+   },
+   
+         "resources":[
+            {
+                "apiVersion":"2015-11-01-preview",
```

Comment:
update apiVersion

Reply:
Updated for this and the other below. Just a note though - the latest api version for this resource type is 2017-03-03-preview (more than a year back).  

Similarly in the prereq.azuredeployment.json, the latest version for workspace resource is "2015-11-01-preview".